mavis/test/mavis_constants.py

```diff
@@ -98,6 +98,23 @@ class PrescreeningQuestion(StrEnum):
     NOT_PREGNANT = "are not pregnant"
 
 
+class ConsentRefusalReason(StrEnum):
+    VACCINE_ALREADY_RECEIVED = "Vaccine already received"
+    VACCINE_WILL_BE_GIVEN_ELSEWHERE = "Vaccine will be given elsewhere"
+    MEDICAL_REASONS = "Medical reasons"
+    PERSONAL_CHOICE = "Personal choice"
+    OTHER = "Other"
+
+    @property
+    def requires_details(self) -> bool:
+        return self is not ConsentRefusalReason.PERSONAL_CHOICE
+
+
+class ConsentMethod(StrEnum):
+    ONLINE = "Online"
```

Comment:
We should actually stop using this option in the tests, as it was never supposed to be there for nurses and will be going away soon: https://nhsd-jira.digital.nhs.uk/browse/MAV-1220

The "Online" option was supposed to represent when the parent submits consent directly via the parental consent flow and should never be used when nurses are recording verbal consent.